sample-app/src/Layout.tsx

```diff
@@ -0,0 +1,29 @@
+import Navigation from './components/Navigation';
+import SearchBar from './components/SearchBar';
+import { useAnswersState } from '@yext/answers-headless-react';
+import { universalResultsConfig } from './universalResultsConfig';
+
+const navLinks = [
+  {
+    to: '/',
+    label: 'All'
+  },
+  ...Object.entries(universalResultsConfig).map(([verticalKey, config]) => ({
+    to: verticalKey,
+    label: config.label || verticalKey
+  }))
+]
+
+export default function Layout({ page }: { page: JSX.Element }) {
```

Comment:
can you add some jsdocs to help describe the responsibility of these different components?

Reply:
added some and also renamed BaseRouter -> PageRouter